src/PlacePicker/__tests__/PlacePicker.test.js

```diff
@@ -0,0 +1,68 @@
+// @flow
+
+import React from 'react';
+import { shallow, render, fireEvent } from 'react-native-testing-library';
+import PlacePicker, { filterSuggestedPlaces } from '../PlacePicker';
+import suggestedPlaces, { Prague, Berlin } from '../mocks/places';
+import { Badge } from '../../Badge';
+
+test('filterSuggestedPlaces', () => {
+  expect(filterSuggestedPlaces([Prague, Berlin], [Berlin])).toEqual([Prague]);
+});
+
+describe('#PlacePicker', () => {
```

Comment:
What for is the `#`? Is it some convention?

Reply:
done